lib/haml_lint/linter/rubocop.rb

```diff
@@ -56,7 +56,12 @@ def find_lints(ruby, source_map)
     # @param file [String]
     # @return [Array<RuboCop::Cop::Offense>]
     def lint_file(rubocop, file)
-      rubocop.run(rubocop_flags << file)
+      status = rubocop.run(rubocop_flags << file)
+      unless [::RuboCop::CLI::STATUS_SUCCESS, ::RuboCop::CLI::STATUS_OFFENSES].include?(status)
+        raise HamlLint::Exceptions::ConfigurationError,
+              "RuboCop exited unsuccessfully with status #{status}." \
+              ' Check the stack trace to see if there was a misconfiguration.'
+      end
```

Comment:
https://github.com/rubocop-hq/rubocop/blob/cc7e221b0e4723a75b59c4bd725a5555a13068dd/lib/rubocop/cli.rb#L7-L10

We may be able to improve the error message if we are considering errors except for misconfiguration. 🤔